File: VistaMenu/Demo/Form1.cs
using System;
using System.Drawing;
using System.Windows.Forms;

namespace VistaMenuDemo
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            Font = SystemFonts.MessageBoxFont;

            InitializeComponent();
            this.ContextMenu = contextMenu;
        }

        private void btnSetOptionsImage_Click(object sender, EventArgs e)
        {
            vistaMenu.SetImage(mnuOptions, vistaMenu.GetImage(mnuAddFiles));
        }

        private void btnClearOptionsImage_Click(object sender, EventArgs e)
        {
            vistaMenu.SetImage(mnuOptions, null);
        }
    }
}

File: VistaMenu/DemoVB/Form1.vb
Public Class Form1

    Private Sub btnSetOptionsImage_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btnSetOptionsImage.Click
        vistaMenu.SetImage(mnuOptions, vistaMenu.GetImage(mnuAddFiles))
    End Sub

    Private Sub btnClearOptionsImage_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btnClearOptionsImage.Click
        vistaMenu.SetImage(mnuOptions, Nothing)
    End Sub

    Private Sub Form1_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
        Me.Font = SystemFonts.MessageBoxFont

        Me.contextMenu = contextMenu
    End Sub
End Class
